cairo/src/utils/signature.cairo

```diff
@@ -57,12 +59,12 @@ namespace Signature {
     // * r is the x coordinate of some nonzero point on the curve.
     // * All the limbs of s and msg_hash are in the range (-2 ** 210.99, 2 ** 210.99).
     // * All the limbs of r are in the range (-2 ** 124.99, 2 ** 124.99).
-    func try_recover_public_key{range_check_ptr}(
-        msg_hash: BigInt3, r: BigInt3, s: BigInt3, v: felt
+    func try_recover_public_key{range_check_ptr, keccak_ptr: KeccakBuiltin*}(
+        msg_hash: BigInt3, r: BigInt3, s: BigInt3, y_parity: felt
     ) -> (public_key_point: EcPoint, success: felt) {
         alloc_locals;
         let (local r_point: EcPoint*) = alloc();
-        let (is_on_curve) = try_get_point_from_x(x=r, v=v, result=r_point);
+        let (is_on_curve) = try_get_point_from_x(x=r, v=y_parity, result=r_point);
```

Comment:
natspect needs to be updated as v is replaced by y_parity

Reply:
natspec of what?